stubs/app/client/ui/NewDocMethods.ts

```diff
@@ -0,0 +1,47 @@
+import {docImport, importFromPlugin} from 'app/client/ui/HomeImports';
+import {docUrl, urlState} from 'app/client/models/gristUrlState';
+import {HomeModel} from 'app/client/models/HomeModel';
+import {ImportSourceElement} from 'app/client/lib/ImportSourceElement';
+import {reportError} from 'app/client/models/AppModel';
+
+export async function createDocAndOpen(home: HomeModel) {
```

Comment:
Adding significant new code to `stubs` is undesirable, unless it is for `grist-core` and only `grist-core`, not desktop, not ee, not saas etc.

Reply:
I do have a doubt that maybe desktop is using some material from stubs? But if so ideally that material would go into `core/` and be used from desktop ext and by `stubs`.

Reply:
Desktop is using `version.ts` from `stubs`. I just removed it. Now there is no `core/stubs` dependency.